src/data_provider/src/packages/macportsWrapper.h

```diff
@@ -31,8 +31,7 @@ class MacportsWrapper final : public IPackageWrapper
 {
     public:
         explicit MacportsWrapper(SQLite::IStatement& stmt)
-            : m_version{UNKNOWN_VALUE}
```

Comment:
I know we discussed this on our daily meeting, but just to mention we took a different approach here 
https://github.com/wazuh/wazuh/pull/18006 
`                    packageJson["version"]      = version.empty() ? UNKNOWN_VALUE : std::move(version);`

Reply:
I worked according to today's daily decision.
Maybe a final confirmation from @Dwordcito should be necessary.

Reply:
Yeah yeah I know that this is what we agreed today, it was just a reminder, not a change request.